packages/manager/modules/pci/src/projects/project/serving/namespace/models/add/add.html

```diff
@@ -0,0 +1,138 @@
+<div class="servingAddModel">
+    <h1 data-translate="pci_projects_project_serving_namespace_models_add"></h1>
+
+    <oui-stepper data-on-finish="$ctrl.addModel()">
+
+        <oui-step-form
+            data-header="{{ :: 'pci_projects_project_serving_namespace_models_add_type' | translate }}"
+            data-editable="!$ctrl.isAdding"
+            data-loading="$ctrl.isAdding">
+            <oui-radio
+                data-ng-repeat="workflowTemplate in $ctrl.workflowTemplates track by $index"
+                data-model="$ctrl.model.workflowTemplate"
+                data-name="workflowTemplate"
+                data-value="workflowTemplate"
+                data-required>
+                <span
+                    data-translate="{{:: 'pci_projects_project_serving_namespace_models_add_type_' + workflowTemplate.toLowerCase() }}"></span>
+                <button
+                    type="button"
+                    class="oui-popover-button"
+                    data-oui-popover="{{:: 'pci_projects_project_serving_namespace_models_add_type_' + workflowTemplate.toLowerCase() + '_popover' | translate }}"
+                    data-oui-popover-placement="auto"></button>
+            </oui-radio>
+        </oui-step-form>
+
+        <oui-step-form
+            data-header="{{ :: 'pci_projects_project_serving_namespace_models_add_model' | translate }}"
+            data-editable="!$ctrl.isAdding"
+            data-valid="$ctrl.model.workflowTemplate == $ctrl.PRESET_IMAGE || $ctrl.model.storagePath"
+            data-loading="$ctrl.isAdding">
+            <div class="container-fluid px-0" data-ng-if="$ctrl.model.workflowTemplate == $ctrl.PRESET_IMAGE">
+                <div class="row">
+                    <div class="col-md-6 col-xl-4 my-3"
+                         data-ng-repeat="preset in $ctrl.presetImages track by preset.id">
+                        <oui-select-picker
+                            data-name="{{:: preset.id }}"
+                            data-description="{{:: preset.description }}"
+                            data-model="$ctrl.model.imageId"
+                            data-label="{{:: preset.name }}"
+                            data-variant="light"
+                            data-values="[preset]"
+                            data-required>
+
+                            <span class="oui-select-picker__section">
+                                <a target="_blank" class="oui-button oui-button_link" rel="noopener"
+                                   data-ng-href="{{:: preset.link }}">
+                                    <span>{{ :: 'pci_projects_project_serving_namespace_models_add_model_preset_info' | translate }}</span>
+                                </a>
```

Comment:
Maybe adding a  `oui-icon-external_link`and the corresponding aria-description could be nice to indicate that this opens a new browser tab